airbyte-cdk/java/airbyte-cdk/core/src/main/kotlin/io/airbyte/cdk/db/jdbc/DateTimeConverter.kt

```diff
@@ -5,17 +5,18 @@ package io.airbyte.cdk.db.jdbc
 
 import com.fasterxml.jackson.databind.node.ObjectNode
 import io.airbyte.cdk.db.DataTypeUtils
+import io.github.oshai.kotlinlogging.KotlinLogging
 import java.sql.*
 import java.time.*
 import java.time.format.DateTimeFormatter
 import java.util.concurrent.*
 import kotlin.math.abs
 import kotlin.math.min
-import org.slf4j.Logger
-import org.slf4j.LoggerFactory
+
+private val LOGGER = KotlinLogging.logger {}
```

Comment:
curious: is it preferable to declare the logger outside the class?

Reply:
Yeah this is a question i have too. I'm inconsistently doing it inside the class and only outside when I need it in companion object.

Reply:
I think it's either outside the class or inside the companion object. Definitely not inside the class, as that would create a new instance for each class instance